consensus/istanbul/core/handler.go

```diff
@@ -193,7 +199,26 @@ func (c *core) handleCheckedMsg(msg *message, src istanbul.Validator) error {
 	return errInvalidMessage
 }
 
-func (c *core) handleTimeoutMsg() {
+func (c *core) handleTimeoutMsg(view *istanbul.View) {
+	// TODO-Klaytn-Istanbul: Check whether this statement is required. EN/PN should not handle consensus msgs.
+	if c.backend.NodeType() != common.CONSENSUSNODE {
+		logger.Warn("Received an istanbul timeout message from EN or PN",
```

Comment:
Isn't this should be like `This is not consensus node, but received an istanbul timeout message`? the log `from EN or PN` doesn't seem quite right.

Reply:
You are right. Thanks!